openreview/arr/process/configuration_process.py

```diff
@@ -0,0 +1,822 @@
+def process(client, note, invitation):
+    import traceback
+    import json
+    from openreview.stages.arr_content import (
+        arr_registration_task_forum,
+        arr_registration_task,
+        arr_content_license_task_forum,
+        arr_content_license_task,
+        arr_reviewer_ac_recognition_task_forum,
+        arr_reviewer_ac_recognition_task,
+        arr_max_load_task_forum,
+        arr_reviewer_max_load_task,
+        arr_ac_max_load_task,
+        arr_sac_max_load_task,
+        arr_reviewer_emergency_load_task_forum,
+        arr_reviewer_emergency_load_task,
+        arr_ac_emergency_load_task_forum,
+        arr_ac_emergency_load_task,
+        arr_reviewer_checklist,
+        arr_ae_checklist,
+        arr_desk_reject_verification,
+        arr_reviewer_consent_content,
+        arr_official_review_content,
+        arr_metareview_content,
+        arr_ethics_review_content,
+        arr_review_rating_content,
+        arr_author_consent_content,
+        arr_metareview_license_task,
+        arr_metareview_license_task_forum,
+        hide_fields_from_public
+    )
+    from openreview.arr.helpers import ARRStage
+    from openreview.venue import matching
+    from datetime import datetime
+
+    client_v2 = openreview.api.OpenReviewClient(
+        baseurl=openreview.tools.get_base_urls(client)[1],
+        token=client.token
+    )
+
+    SUPPORT_GROUP = invitation.id.split('/-/')[0]
+    invitation_type = invitation.id.split('/')[-1]
+    forum_note = client.get_note(note.forum)
+
+    comment_readers = [forum_note.content.get('venue_id') + '/Program_Chairs', SUPPORT_GROUP]
+    venue_id = forum_note.content.get('venue_id')
+    domain = client_v2.get_group(venue_id)
+    meta_invitation_id = domain.content['meta_invitation_id']['value']
+    request_form_id = domain.content['request_form_id']['value']
+    request_form = client.get_note(request_form_id)
+    support_group = request_form.invitation.split('/-/')[0]
+    venue = openreview.helpers.get_conference(client, request_form_id, support_group)
+    invitation_builder = openreview.arr.InvitationBuilder(venue)
+    
+    request_form = client.get_note(request_form_id)
+    support_group = request_form.invitation.split('/-/')[0]
+    venue_stage_invitations = client.get_all_invitations(regex=f"{support_group}/-/Request{request_form.number}.*")
+    venue = openreview.helpers.get_conference(client, request_form_id, support_group)
+    invitation_builder = openreview.arr.InvitationBuilder(venue)
+
+    # Set stages
+    def build_preprint_release_edit(client, venue, builder, request_form):
```

Comment:
I wonder if we should move this code to the ARR class, it will be easier to maintain and fix bugs. 

You also have access to the committee names and invitations names so you don't have to have anything hardcoded.